containers/org.eclipse.linuxtools.docker.ui/src/org/eclipse/linuxtools/docker/ui/launch/ContainerLauncher.java

```diff
@@ -2048,20 +1460,8 @@ private void setConsole(RunConsole cons) {
 	 *
 	 * @since 3.0
 	 */
-	public Set<String> getCopiedVolumes(String connectionName,
-			String imageName) {
-		Set<String> copiedSet = new HashSet<>();
-		if (copiedVolumesMap != null) {
-			Map<String, Set<String>> connectionMap = copiedVolumesMap
-					.get(connectionName);
-			if (connectionMap != null) {
-				Set<String> imageSet = connectionMap.get(imageName);
-				if (imageSet != null) {
-					copiedSet = imageSet;
-				}
-			}
-		}
-		return copiedSet;
+	public static Set<IPath> getCopiedVolumes(IPath hostdir) {
```

Comment:
The comment describing this new method is actually for the previous method.  You need to add a new description.